ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -121,8 +122,9 @@
    * |     PUT          |           N                     |       N - 1      |
    * |    DELETE        |          3~N                    |         2        |
    * |   TTLUpdate      |          3~N                    |         2        |
+   * |   UNDELETE       |          3~N                    |         2        |
```

Comment:
According to the design doc, the quorum target for the UNDELETE should be 2 per DC. Is there a follow up PR for this?